apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryClient.DatasetCrud.cs

```diff
@@ -256,48 +243,44 @@ public virtual BigQueryDataset CreateDataset(DatasetReference datasetReference,
             GetOrCreateDataset(GetDatasetReference(projectId, datasetId), getOptions, createOptions);
-
+        
         /// <summary>
         /// Attempts to fetch the specified dataset, creating it if it doesn't exist.
         /// </summary>
         /// <param name="datasetReference">A fully-qualified identifier for the dataset. Must not be null.</param>
         /// <param name="getOptions">The options for the "get" operation. May be null, in which case defaults will be supplied.</param>
         /// <param name="createOptions">The options for the "create" operation. May be null, in which case defaults will be supplied.</param>
         /// <returns>The existing or new dataset.</returns>
-        public virtual BigQueryDataset GetOrCreateDataset(DatasetReference datasetReference, GetDatasetOptions getOptions = null, CreateDatasetOptions createOptions = null)
-        {
+        public virtual BigQueryDataset GetOrCreateDataset(DatasetReference datasetReference, GetDatasetOptions getOptions = null, CreateDatasetOptions createOptions = null) =>
             throw new NotImplementedException();
-        }
-
+        
         /// <summary>
         /// Asynchronously attempts to fetch the specified dataset within this client's project, creating it if it doesn't exist.
-        /// This method just creates a <see cref="DatasetReference"/> and delegates to <see cref="GetOrCreateDatasetAsync(DatasetReference,GetDatasetOptions,CreateDatasetOptions,CancellationToken)"/>.
+        /// This method just creates a <see cref="DatasetReference"/> and delegates to <see cref="GetOrCreateDatasetAsync(DatasetReference, GetDatasetOptions, CreateDatasetOptions, CancellationToken)"/>.
         /// </summary>
         /// <param name="datasetId">The dataset ID. Must not be null.</param>
         /// <param name="getOptions">The options for the "get" operation. May be null, in which case defaults will be supplied.</param>
         /// <param name="createOptions">The options for the "create" operation. May be null, in which case defaults will be supplied.</param>
         /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
         /// <returns>A task representing the asynchronous operation. When complete, the result is
         /// the existing or new dataset.</returns>
-        public virtual Task<BigQueryDataset> GetOrCreateDatasetAsync(string datasetId,
-            GetDatasetOptions getOptions = null, CreateDatasetOptions createOptions = null, CancellationToken cancellationToken = default(CancellationToken)) =>
+        public virtual Task<BigQueryDataset> GetOrCreateDatasetAsync(string datasetId, GetDatasetOptions getOptions = null, CreateDatasetOptions createOptions = null, CancellationToken cancellationToken = default(CancellationToken)) =>
             GetOrCreateDatasetAsync(GetDatasetReference(datasetId), getOptions, createOptions, cancellationToken);
-
+        
```

Comment:
That's just because we didn't have any spaces on some blank lines before. We now add spaces for everything when we indent. I'm not bothered, personally :) (A lot of our code already has lines like this.)